src/features/remote-control-menu/remote-control-menu.js

```diff
@@ -106,4 +121,24 @@ export default class RemoteControlMenu extends React.Component {
     _onSetVideoMute() {
         remoteControlService.setVideoMute(!this.props.videoMuted);
     }
+
+    /**
+     * Changes the current wireless screensharing state.
+     *
+     * @private
+     * @returns {void}
+     */
+    _onSetWirelessScreensharing() {
```

Comment:
this says `set` but really it's a toggle, no?

Reply:
You are right. I think I followed the naming convention I had in this file previously, where everything was set for some reason. I can change this instance and change the others in a future pr.